test/test_autograd.py

```diff
@@ -21,6 +21,9 @@
 # Autograd tests use double as the default dtype
 torch.set_default_dtype(torch.double)
 
+# TODO(alband) Remove this when this flag is not needed anymore
+torch._C._set_forward_AD_enabled(True)
+
```

Comment:
One question about the autogenerated code samples in the PR message: I see some helper functions named “toLegacyFwGrad” / “toLegacyPrimal” — what is “legacy” about them?

Reply:
I did a refactor before this PR to remove these. So they are not in the latest version of the PR. Let me update the description.

Reply:
Updated.